.github/TemplateIssueBug.md

```diff
@@ -0,0 +1,29 @@
+### What happens?
+…
+
+
+### What were you expecting to happen?
+…
+
+
+### What are the steps to reproduce this issue?
+1. _Go to '…'_
+2. _Click on '…'_
+3. _Scroll down to '…'_
```

Comment:
code snippets will be necessary (more like it)

ask for a smallest reproducible example either as inline code, gist or a Pull Request/separate repo (cpp code listing).